catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -66,9 +71,14 @@ public EnumerationValidator(final Set<String> values) {
     public Optional<AttributeValidationReport> validate(final Attribute attribute) {
```

Comment:
Oops, I think I meant the javadoc for this `validate()` method. It says that comparisons are case-sensitive.